src/CommunityToolkit.Maui.Core/Views/DrawingView/Handlers/DrawingViewHandler.shared.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.Maui.Handlers;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// DrawingView handler
+/// </summary>
+public partial class DrawingViewHandler
```

Comment:
Friendly reminder to implement the Mappers. Without it, this control will not work in other Frameworks like `Comet`

Reply:
But I do not use any mappers. As I understand they are needed for controls which we cannot modify like Android button. But here we have custom DrawingNativeView control which receives IDrawingView in ctor. Maybe it is not correct. And I shouldn't reference IDrawingView in DrawingNativeView?

Reply:
Take look at the Popup, I use the same approach (create another control that receives the interface in the ctor)

Reply:
updated